### DISCUSSION TITLE
Next.js로 기술 블로그 만들기: 블로그는 사드세요... 제발 3편 내 블로그 간지나게 자랑하기 (Open Graph 적용하기) - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#43] [Announcements]
# Next.js로 기술 블로그 만들기: 블로그는 사드세요... 제발 3편 내 블로그 간지나게 자랑하기 (Open Graph 적용하기) - 정현수 기술 블로그

이건 내 블로그고, 이건 내 Open Graph야.

https://junghyeonsu.com/posts/please-buy-the-blog-3/

**Comment by 6mini** [comment]
블로그 포스팅 정말 잘봤습니다 :)
평소 깃허브를 꾸준히 운영하다, 커스터마이징에 대한 갈증 때문에 직접 개볼해볼까 고민을 하다가
직접 블로그 개발하고 꾸준히 운영하는 분 많이 없어서 더욱 망설였는데, 덕분에 도전해보렵니다 ㅎㅎ
비효율적이지만 저는 가볍게 플라스크로 도전해보겠습니다 ㅎㅎ
